pkgs/swiftgen/swift2objc/lib/src/generator/_core/utils.dart

```diff
@@ -12,3 +14,38 @@ String generateParameters(List<Parameter> params) {
     return "${labels}: ${param.type.name}";
   }).join(", ");
 }
+
+extension Indentation on String {
+  String indent([int count = 1]) {
+    assert(count > 0);
+    final lines = split("\n");
+    final indentation = List.filled(count, "  ").join();
+    return lines.map((line) => "$indentation$line").join("\n");
+  }
+}
+
+extension PathSegments on String {
```

Comment:
Use `package:path` rather than writing this logic yourself. It can be tricky to get all the details right.

Reply:
done